pyproject.toml

```diff
@@ -32,7 +32,7 @@ ipywidgets = "8.1.2"
 # If updating setuptools version, please verify the odh-notebooks-sync workflow continues to work.
 # https://github.com/project-codeflare/codeflare-sdk/actions/workflows/odh-notebooks-sync.yml
 # Reference: https://github.com/target/strelka/pull/469
-setuptools = "<= 70.1.0"
+setuptools = "<= 73.0.0"
```

Comment:
Can you share the reason for the restriction on the value ? Would like to learn about it.

Reply:
I am not too sure @ChristianZaccaria might know more, he should be back tomorrow. This is likely due to the referenced Pull Request above the package.